# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/zdoom/hotfix-cmake-policy.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ./CreateLaunchers.cmake.vanilla	2025-12-31 17:35:03.679969947 +0000
+++ ./CreateLaunchers.cmake	2025-12-31 17:35:22.473205902 +0000
@@ -45,7 +45,7 @@
 set(__create_launchers YES)
 
 if( POLICY CMP0026 )
-	cmake_policy( SET CMP0026 OLD )
+	cmake_policy( SET CMP0026 NEW )
 endif( POLICY CMP0026 )
 
 include(CleanDirectoryList)
